internals/handlers/handlers.go

```diff
@@ -180,6 +180,41 @@ func RenderKeyHandler(c *gin.Context) {
 	c.Data(status, "application/octet-stream", keyResult)
 }
 
+var catMap = map[int]string{
```

Comment:
Move both catMap and langMap to utils and rename them as CategoryMap and vice versa. Call in handlers.go with utils.LanguageMap

Reply:
Make sure both are at the top of file after types under handlers.go , i.e. before function definitions